common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -299,7 +305,7 @@ def is_user_authenticated(self, context):
         # NOTE (CCB): We default to true to maintain the behavior in place prior to allowing anonymous access access.
         return context.get('user_authenticated', True)
 
-    def _student_view(self, context, banner_text=None):
+    def _student_view(self, context, banner_text=None, view_type=STUDENT_VIEW):
```

Comment:
I think `view_type` should be changed to `view` everywhere in this PR (unless it is conflicting with anything). It is not clear what `type` refers to.